slides-export/src/main/groovy/org/asciidoctor/gradle/slides/export/base/AbstractExportBaseTask.groovy

```diff
@@ -280,10 +277,8 @@ abstract class AbstractExportBaseTask extends DefaultTask {
         dependsOn tasks.findAll {
             if (it instanceof Task) {
                 true
-            } else if (GRADLE_GE_4_8) {
-                it instanceof org.gradle.api.tasks.TaskProvider
             } else {
-                false
+                it instanceof org.gradle.api.tasks.TaskProvider
```

Comment:
We can now replace this if-else with `findAll { it instanceof Task || it instanceof TaskProvider }`.

No need for the `CompileDynamic` either as it was only required for compatibility with Gradle 4.7 and older.

Reply:
c0486193b8616dc366eaa3ea1e55cc6729288ccc